CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ set(CMAKE_C_FLAGS_DEBUG   "-g -O0")
 
 set(CALL_TARGET_BINS_DIRECTLY ON CACHE BOOL "Enable calling sysrepoctl/sysrepocfg directly at install time")
 set(INDIRECT_YANG_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/yang/" CACHE PATH "Where to store YANG files when not executing sysrepoctl directly")
+set(CALL_SYSREPOCTL_BIN "${CMAKE_BINARY_DIR}/src/sysrepoctl" CACHE PATH "Path to the sysrepoctl binary to be used at install time")
+set(CALL_SYSREPOCFG_BIN "${CMAKE_BINARY_DIR}/src/sysrepoctl" CACHE PATH "Path to the sysrepocfg binary to be used at install time")
```

Comment:
You made a typo, this must be sysrepocfg binary, not sysrepoctl. That is why the tests failed.